docs/learn/multiversx-ecosystem.md

```diff
@@ -0,0 +1,59 @@
+
+By employing [_sharding_](/docs/lea), a method of parallelizing data & transactions processing, MultiversX’s performance will scale up with the number of computers joining the network, reaching more than 100.000 transactions per second while growing increasingly decentralized.
+
+:::important
+**What is a blockchain?**
+
+A blockchain is a database of transactions that is continually updated and shared across numerous computers in a network. Each new set of transactions is grouped into a "block," which is added to the chain of previous transactions, hence the name blockchain. Public blockchains like MultiversX enable anyone to add data, but not remove it (**immutable**). To alter any information or manipulate the system, one would need to compromise the majority of computers on the network, which is an incredibly challenging task. This inherent difficulty makes decentralized blockchains like MultiversX highly secure and trustworthy.
+:::
+
+[comment]: # (mx-context-auto)
+
+## **How is MultiversX different?**
+
+The MultiversX network is the first to present a viable solution where all three aspects of sharding - state, network, and transactions - have been implemented at once. Combined with its “Adaptive” component, this novel architecture allows for dynamic network configuration to maintain a high level of security while scaling with demand.
+
+In addition to scaling through sharding, MultiversX also approaches the consensus problem with a mechanism called Secure Proof of Stake, which mitigates potential attack vectors when compared to Proof of Work, while also enabling large throughput and fast execution.
+
+By solving some of the hardest consensus and sharding problems in the blockchain space, MultiversX is able to provide a very high level of performance on a network made of inexpensive computers, resulting in a very low cost per transaction. In addition to performance and cost, MultiversX also stands out through the quality of the developer experience and the resulting boost in usability on the end-user side.
+
+[comment]: # (mx-context-auto)
+
+## **Why would I use MultiversX?**
+
+MultiversX is a collaborative story written by all of us, inviting any builder to discover the digital universes that can be built together no matter that it is in DeFi, Gaming or Payments area. MultiversX nodes are distributed to more than 125 different staking providers, ensuring a robust and secure network. It becomes invaluable for individuals facing uncertainty regarding the security, stability, or mobility of their assets due to external forces beyond their control, and thanks to the ESDT standard and guardians that enhance its security.
```

Comment:
"it is in DeFi, Gaming or Payments area. MultiversX nodes are distributed to more than 125 different staking providers"
maybe we should also mention (maybe with a count) the number of individual nodes/identities that are not SPs